service-commands/scripts/init-repos.sh

```diff
@@ -32,18 +33,17 @@ npm install &
 # setup creator node
 cd $PROTOCOL_DIR/
 cd creator-node/
-npm install &
-npm install & #why does it not work without this?
+npm install
 
 # setup libs
 cd $PROTOCOL_DIR/
 cd libs/
-npm install &
+npm install
 
 # setup identity service
 cd $PROTOCOL_DIR/
 cd identity-service/
-npm install --dev &
+npm install --also=dev
 
 wait
```

Comment:
The code above this is synchronous, we don't need this, right?

Reply:
CHERAN YOU'RE ON VACATION also this was added in response to a warning in npm install that said `npm install --dev is deprecated; use npm install --also=dev instead`